lms/djangoapps/learner_dashboard/api/utils.py

```diff
@@ -32,3 +41,37 @@ def get_personalized_course_recommendations(user_id):
         log.warning(f'Cannot get recommendations from Amplitude: {ex}')
 
     return True, False, []
+
+
+def get_algolia_courses_recommendation(course_data):
+    """ Get personalized courses recommendation from Algolia search. """
```

Comment:
Can we add a small description to the doc string on what the algorithm looks like? 
- Input
- What the output will be?
- What ranking is used for search results.